.all-contributorsrc

```diff
@@ -254,7 +254,8 @@
       "contributions": [
         "ideas",
         "doc",
-        "review"
+        "review",
```

Comment:
Oh no @BrainonSilicon - I think rebasing the branch accidentally committed your changes!

Reply:
I think the problem is that `WIP-Inclusive-Events-chapter-Content` (the branch that this PR aims to merge from) was rebased at some point to pull more commits from `main`, while the branch that's the target of this PR (`WIP]Inclusive-Events-chapter`) is further behind `main` in comparison. 

I'm not sure what the best practice would in such a case. Would it be to rebase both PRs to be on the same level of the `main` branch?

Reply:
Massive thanks to @gedankenstuecke and @da5nsy for figuring it out (and @JimMadge for the Friday knowledge drop on Slack to guide the way)

Some learnings 
- PRs work to specific commits and rebases seems to undo this in some way (so be careful when rebasing) 
- best to add references in random places